### PR TITLE
feat(core): Implement parsing for miner transaction

### LINKED ISSUE #407: Implement parsing for miner transaction

### PR DESCRIPTION
**Pull request by dauTT**
Closes #407

### CODE REVIEW

File: packages/neon-core/src/u/StringStream.ts [hunk 1]
```diff
@@ -83,6 +83,10 @@ export class StringStream {
     return len;
   }
 
+  public readUint32(): number {
```
Comment (snowypowers) [inline packages/neon-core/src/u/StringStream.ts]: let's not extend the public interface of StringStream in this PR. We can open another PR for this if there are multiple use cases within the repository.

**Comment by dauTT** [comment]
Running the Integration tests locally are not deterministic. Sometimes some are failing (and they are not the same from one execution to another) and sometimes all are passing. I am not sure what happened after the merge.

**Comment by snowypowers** [comment]
The integration tests has always been a little wonky due to us relying on testnet for it.

**Comment by hal0x2328** [comment]
300 points dauTT
100 points snowypowers for review

**Comment by lllwvlvwlll** [comment]
https://neoscan.io/transaction/c382c87ce7282664dc849fef5cb8c8de22cf3b596dd1b4e85c87efb194bb2003

**Pull request merged**